apps/nycbus/nyc_bus.star

```diff
@@ -0,0 +1,159 @@
+"""
+Applet: NYC Bus
+Summary: NYC Bus departures
+Description: Real-time bus departures for your preferred stop.
+Author: samandmoore
+"""
+
+load("cache.star", "cache")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+load("time.star", "time")
+
+API_KEY = "<INSERT_API_KEY_HERE>"
```

Comment:
I would like to use the secret/encrypt schema stuff once that's ready.

Reply:
Here is the secret docs: https://github.com/tidbyt/pixlet/blob/main/docs/modules.md#pixlet-module-secret

Reply:
@betterengineering hey! the secret docs say to use `app name` with `pixlet encrypt`. In this case, is that "NYC Bus" or should I use the app id "nyc-bus"? I'm just worried that since the app name is two words that might be an issue?

Reply:
It seems like it does the right thing if is just quote it `pixlet "NYC Bus" secret-stuff-here`

Reply:
@betterengineering can you clarify the question @samandmoore asked about the secret encryption?

The [doc](https://github.com/tidbyt/pixlet/blob/main/docs/authoring_apps.md#secrets) says to use `my-app-name` when the example is more like the lower kebab-case `ID` in the manifest and not the `Name`. `make app` doesn't even let you create an app name like that, it tells you to use title case.

`pixlet encrypt` will let me use anything there but since `decrypt` doesn't work locally I can't know if it's correct until it's in the app. It would be helpful if the documentation specified exactly what that should be if it maps directly to something in the manifest or if it's some other obvious value.

Thanks!

Reply:
Got it. Let me re-encrypt the value. 

I used `make app` and didn't have any issues with "NYC Bus" as the app name. I chose that name based on the names of the built in apps. In fact, I also chose the description the same way. So, if the app name and description aren't valid then I guess the built-in apps aren't either? 

I'm totally down to change things if you'd prefer. I mostly just wanted to make this app feel like it fits right in with the "NYC Subway" and "NYC Ferry" apps y'all provide.

Reply:
Okay! all done. 

I reencrypted the value and also just changed the description to remove the "-" so that the validation logic doesn't fail on the hyphenated starting word's conversion to title case.

Reply:
How do you know if it's right? You can type `pixlet encrypt "any-old Thing" secret` and it'll give you a value that you can paste into your app. If only the server can decrypt it and that third parameter is important in the process, then I feel the documentation is a little unclear. I have no way of knowing what other apps used for `pixlet encrypt` so I can't use them as an example.

I know things are new and tools are moving fast, and that's awesome; the last few pixlet releases added a ton of useful things. But I still have a good amount of uncertainty that what I do locally will work on the server and in the mobile app.

Reply:
@noahcolvin i actually don't know if it's right 😅 

but after re-reading the docs, I'm thinking that the id makes more sense than the app name. I'm still hoping that @betterengineering can clarify for us

Reply:
@noahcolvin it appears, from reading the source code, that the "app name" should be the [filename minus the ".star" suffix](https://github.com/tidbyt/pixlet/blob/eb3b928560c732708cb35de63d687dfaf702cd26/runtime/secret.go#L97) ([and here](https://github.com/tidbyt/pixlet/blob/eb3b928560c732708cb35de63d687dfaf702cd26/runtime/secret.go#L54)) but that seems like it wouldn't work properly if the filename is `nyc_bus.star` and the app id is `nyc-bus`. 

Sooo, i'm pretty sure i need to reencrypt this value using `nyc_bus`

Reply:
@samandmoore yeah I saw that last night too but wasn't 💯 sure if that was the case or not. I'd prefer @betterengineering (or anyone who knows) give a definitive answer and a line added to the docs or the example updated to be more clear.

It really killed my momentum last night when I set down to make something and the first thing I tried to do was encrypt an API key and the example didn't really make sense.